packages/console/src/components/CodeEditor/index.module.scss

```diff
@@ -2,4 +2,20 @@
 
 .editor {
   margin: _.unit(1) 0;
+  border-radius: _.unit(4);
+  padding: _.unit(4) 0;
+  // Use fixed color for both light and dark mode, the same as vs-dark.
+  background: #1e1e1e;
```

Comment:
Maybe we can declare a color constant and both light and dark modes use the same color.
Just make the color definition consistent.

Reply:
Is a very special case, this MUST be the same as monaco-editor's theme color.

Reply:
It make no sense to change this color elsewhere, and will not be used by any other component.

Reply:
More like svg's inside color.